go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/gomodule/redigo v2.0.0+incompatible
 	github.com/jarcoal/httpmock v1.3.1
 	github.com/korovkin/limiter v0.0.0-20230307205149-3d4b2b34c99d
+	github.com/libsv/bitcoin-hc v0.3.0
```

Comment:
eeeeeeeeeeee why?

Reply:
I'm not sure if this is good idea,
we're bringing a lot of other dependencies, just to have a 2 structs or something like that.

Reply:
Should I make our own structure and put it in box-models?